lib/travis/guest-api/cache.rb

```diff
@@ -60,6 +63,27 @@ def get_result(job_id)
       return passed ? 'passed' : 'failed'
     end
 
+    def get_added_step_metrics(job_id, class_name)
+      job_record = get_job(job_id)
+      raise AddStepException,
+        "Invalid cache content class_name: #{class_name} couldn\'t be searched" if job_record.nil?
```

Comment:
What do you think about this?:
"Test case #{class_name} could not be found for job id: #{job_id}"